ui/job-view/pushes/FuzzyJobFinder.jsx

```diff
@@ -0,0 +1,293 @@
+/* eslint react/no-multi-comp: 0, react/prop-types: 0 */
```

Comment:
(These eslint-disable lines will need to be removed/fixed before final review)

Reply:
Done.